src/main/java/uk/gov/justice/hmpps/casenotes/dto/CaseNoteType.java

```diff
@@ -30,7 +30,10 @@
     @Builder.Default
     private String activeFlag = "Y";
 
-    @ApiModelProperty(value = "List of case note sub types", position = 4, allowEmptyValue = true)
+    @ApiModelProperty(value = "Source of Case Note Type, legacy case note are null", example = "OCNS", position = 4)
+    private String source;
+
+    @ApiModelProperty(value = "List of case note sub types", position = 5, allowEmptyValue = true)
```

Comment:
As a matter of interest what value does `position` give for Swagger?

Reply:
The order the json exmples are displayed in

Reply:
Yes at the moment, could have been a boolean, instead but maybe other sources in the future?  YAGNI?